web/themes/custom/jcc_components/templates/paragraph/paragraph--hero.html.twig

```diff
@@ -40,12 +40,12 @@
 #}
 
 {% set parent_type = paragraph.getParentEntity().getType() %}
-{% set variant = paragraph.field_variant.value|json_decode %}
```

Comment:
This shouldn't be removed, so `variant.hero` can still be used.

Reply:
These are actually different values. The `variant` isn't related to the hero itself and actually throws a WSOD when you try to access it. Since the other value was working, I left it as-is (but obviously swapped the keys around).

Reply:
How is `variant` not related? Unless the `field_variant` is going to be removed in favor of the content dictating the layout, it is still needed so the user can manually choose.